libr/anal/meson.build

```diff
@@ -141,7 +141,7 @@ r_anal_dep = declare_dependency(link_with: r_anal,
 
 pkgconfig_mod.generate(
   libraries: [r_anal],
-  subdirs: 'libr',
+  subdirs: ['.', 'libr'],
```

Comment:
Why is `.` necessary?

Reply:
Because sdb headers are now (with this PR and the one in sdb) in include/sdb and not in include/libr/sdb. Radare code includes those files with <sdb/xxxx.h> so you need to also include the base include directory. You don’t notice the problem when you install it in a place where the compiler automatically looks for headers, but if you install radare in a different directory (as in Travis) you have an error

Reply:
Alright.